puppet/modules/quickstack/manifests/pacemaker/ceilometer.pp

```diff
@@ -27,6 +29,19 @@
       $_ensure = undef
     }
 
+    if $coordination_backend == 'redis' {
+      $_backend_ips = map_params("lb_backend_server_addrs")
+      $redis_host = $_backend_ips[0]
+      $coordination_url = "redis://${redis_host}:${coordination_backend_port}"
```

Comment:
Now, I don't know redis at all, but it seems to me like this should be something more similar to how we currently point at a list of memcached servers.  Is there a reason we don't want that?  If so, why is the first one in the list of servers always correct([0])?